frontend/src/pages/projects/screens/spawner/service.ts

```diff
@@ -50,18 +48,13 @@ export const createPvcDataForNotebook = async (
 };
 
 export const updatePvcDataForNotebook = async (
-  projectName: string,
   storageData: StorageData,
   existingPvc: PersistentVolumeClaimKind | undefined,
-  dryRun?: boolean,
 ): Promise<{ volumes: Volume[]; volumeMounts: VolumeMount[] }> => {
   const volumes: Volume[] = [];
   const volumeMounts: VolumeMount[] = [];
 
   if (existingPvc && storageData.storageType === StorageType.EXISTING_PVC) {
-    if (isPvcUpdateRequired(existingPvc, storageData)) {
-      await updatePvc(storageData, existingPvc, projectName, { dryRun });
-    }
```

Comment:
Removed these lines, since current implementation does not change the existingPVC data , its just a dropdown to select